src/core/public/ui_settings/__snapshots__/ui_settings_api.test.ts.snap

```diff
@@ -1,126 +1,114 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`#batchSet Buffers are always clear of previously buffered changes: two requests, second only sends bar, not foo 1`] = `
-Object {
-  "matched": Array [
-    Array [
-      "/foo/bar/api/kibana/settings",
-      Object {
-        "body": "{\\"changes\\":{\\"foo\\":\\"bar\\"}}",
-        "credentials": "same-origin",
-        "headers": Object {
-          "accept": "application/json",
-          "content-type": "application/json",
-          "kbn-version": "v9.9.9",
-        },
-        "method": "POST",
```

Comment:
fetch mock `calls()` changed format from returning an object `{matched: [], unmatched: []}` to only returning the matched calls: https://github.com/wheresrhys/fetch-mock/issues/293

Reply:
cc @spalger